src/useChunk.js

```diff
@@ -44,31 +38,10 @@ export function useChunk(defaultContent, { identifier, type }) {
     };
   }
 
-  const fallbackChunk = chunkFallback.filter(chunkItem => chunkItem.identifier === identifier)[0];
+  const fallbackChunk = defaultChunks.filter(chunkItem => chunkItem.identifier === identifier)[0];
```

Comment:
You can use `defaultChunks.find(...)`, which will return early on the first match!

Also take a look at [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo), which can speed up lookups when the array is sufficiently large.

(If you use a hook, that'll mean moving this line _above_ the early return (`if (error)`)